lib/naughty/core.lua

```diff
@@ -583,6 +583,7 @@ function naughty.notify(args)
             die(naughty.notificationClosedReason.dismissedByUser)
         end
     end
+    notification.run = run
```

Comment:
Uhm... that makes me wonder "why is `naughty.notifications` accessible?" That makes a terrible API since it just exposes internals. @Elv13 Opinions?